au4/src/projectscene/qml/Audacity/ProjectScene/clipsview/ClipItem.qml

```diff
@@ -16,6 +16,9 @@ RoundedRectangle {
     property color clipColor: "#677CE4"
     property bool clipSelected: false
 
+    property real dragMaximumX: 100000
```

Comment:
Any reason behind that particular number? Or is it just for now to be able to freely drag clips pretty much everywhere?

Reply:
it's just a deliberately large number, the distance we can drag the rightmost clip, which has no limit (the real limit is the screen size)